TTS/tts/datasets/dataset_downloaders.py

```diff
@@ -0,0 +1,256 @@
+import logging
```

Comment:
Can you separate this to a new PR ? 

You should also create separate PRs for changes under 🐸TTS as we move AutoTTS to a new repo

Reply:
yea I had made this before I you guys added the downloaders so I'm gong to make a new PR just adding functions for the other datasets